src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -64,4 +66,15 @@ def create_branched_package
 
     target_package
   end
+
+  def write_report
+    return unless workflow_run = WorkflowRun.find(workflow_run_id)
+
+    # step_key = :branch_package
+    step_key = Workflow::SUPPORTED_STEPS.key(self.class).to_s
+    report = workflow_run.report ? JSON.parse(workflow_run.report) : {}
+    report[step_key] ||= []
+    report[step_key] << { 'source_project' => source_project_name, 'source_package' => source_package_name, 'target_project' => target_project_name, 'target_package' => target_package_name }
+    workflow_run.update(report: report.to_json)
```

Comment:
If you store json, why don't you [serialize](https://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Serialization/ClassMethods.html) it?

Reply:
Sorry, I stashed that change by mistake.

Reply:
Included in my last push now: https://github.com/openSUSE/open-build-service/compare/b73b7db0ad4b62124223778927f89056fced568e..428c6fb5e0ab662dac76b84dfe6d5893da46d0d8